core/crypto/crypto_core.cpp

```diff
@@ -32,6 +32,7 @@
 
 #include "core/os/os.h"
 
+#define MBEDTLS_ALLOW_PRIVATE_ACCESS
```

Comment:
Why is this needed?

Reply:
To have access to `mbedtls_ecp_keypair` struct members.